### ISSUE TITLE
[Issue] System Tray menus are missing important styles.

### ISSUE THREAD

**Issue by Cu3PO42** [end-4/dots-hyprland#512]
- **I have read the [Usage](https://end-4.github.io/dots-hyprland-wiki/en/i-i/02usage) and the [Troubleshooting](https://end-4.github.io/dots-hyprland-wiki/en/i-i/03troubleshooting) pages of the wiki**
- **I have made sure that both my config and system packages are up to date**
- Linux distro: NixOS

---
## The issue

The CSS rules applied to context menus in the system tray hide important details. For example, the context menu of nm-applet currently looks like this:

![image](https://github.com/end-4/dots-hyprland/assets/5256300/a3265309-cdf5-45e8-842f-e766c90fa7b1)

Available networks should have an arrow or other indicator that it has a sub-menu and "Enable Wi-Fi" and "Enable Networking" should both have checkboxes. If I remove any custom CSS they do:

![image](https://github.com/end-4/dots-hyprland/assets/5256300/a8ad1dc8-216e-44e1-beb2-14fe536a41ad)

Additionally, the icon for nm-applet is very different with styles applied. I'll tr to investigate why this happens and how it may be fixed.

**Comment by end-4** [comment]
caused by `* {all: unset}` for sure
haven't found out how to keep styles for those stuff

**Comment by Cu3PO42** [comment]
Agreed. I was thinking of trying to do `*:not(....) } {}`, but I'm not yet sure how to select the menu. And GTK inspector also isn't particularly helpful because the menu doesn't stay open as soon as I click over to the inspector.

A cleaner solution would likely be to add back the necessary rules to show dividers, checkboxes and arrows.

**Comment by end-4** [comment]
this is irrelevant now that we use quickshell instead of ags
